frontend/src/components/feed/ContentList.tsx

```
@@ -0,0 +1,174 @@
+          });
+          break;
+        case "trending":
+          response = await axios.get(`${apiURL}/content/feed/trending`, {
+            timeout: 5000,
+          });
+          break;
+        default:
+          return false;
+      }
+
+      if (response.data && response.data.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)